### DISCUSSION TITLE
I am about to throw Supabase out the window and write my own even though I love the idea...

### DISCUSSION THREAD

**Discussion by GQAdonis** [supabase/supabase#9163] [Questions]
I love the idea of Supabase as a product, but I must say it is very poorly implemented and documented for self-hosting.  This is especially true using Kubernetes.  It is a difficult thing to get working in production with real SSL certs, etc.

But...I actually got it done.  After two days I am staring at a screen that seeks to have me login using my Github ID, but when I try (even after configuring GoTrue for Github AND Google), I get a 404.  I can never see my project or administer the data.

Can anyone help?

**Comment by Ziinc** [comment]
Hi @GQAdonis we would need more information on your setup in order to provide advice on how to proceed, due to how customized and opinionated each self hosted setup is like.
The GoTrue server documentation is [here](https://supabase.com/docs/gotrue/server/about), perhaps you are missing a configuration value for your external provider
